tool/augment.js

```diff
@@ -155,6 +155,12 @@ module.exports = {
             help: 'Random/ Uniform strategy assigns random/ uniform weights to parameters instead of reading from file',
             default: false
         });
+        parser.add_argument('--subset-param-set', {
+            required: false,
+            type: Array,
```

Comment:
This type notation is incorrect. `type` is a function that takes each value and returns the parse. You're passing one string value, and converting to array, you will get one element. Did you test this?

Reply:
Ok, I changed it so we that we pass a string to augment-worker, and then parse it to an array before passing to the dataset augmenter.

Reply:
That's fine